tools/bf_autocorres.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python2
+
```

Comment:
It is a very valid shebang line. Try to execute `/usr/bin/env python2` locally, and read: http://en.wikipedia.org/wiki/Shebang_%28Unix%29#Syntax